src/components/helpdesk/repo/problem.repo.js

```diff
@@ -0,0 +1,24 @@
+const problemModel = require('../../../services/helpdesk/model');
+const { deleteManyResult } = require('../../helper/repo.helper');
+
+/**
+ * Return helpdesk problems for userId
+ * @param {String|ObjectId} userId
+ */
+const getProblemsForUser = async (userId) => {
+	return problemModel.find({ userId }).lean().exec();
+};
+
+/**
+ * Removes all helpdesk problems for userId
+ * @param {String|ObjectId} userId
+ */
+const deleteProblemsForUser = async (userId) => {
+	const deleteResult = await problemModel.deleteMany({ userId }).lean().exec();
```

Comment:
what happens if userId is not set here? we should ensure not to delete all problems not having a userId